modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -4,20 +4,22 @@
 
 [id="installation-special-config-encrypt-disk-tang_{context}"]
 = Enabling Tang disk encryption
-Use this procedure to enable Tang mode disk encryption during {product-title} deployment.
+Use this procedure to enable Tang mode disk encryption during an {product-title} installation.
```

Comment:
```suggestion
Use the following procedure to enable Tang mode disk encryption during an {product-title} installation.
```

Reply:
@sagidlow thank you for your review! I have now resolved this suggestion.